zenodo/modules/records/serializers/schemas/dc.py

```diff
@@ -30,6 +30,25 @@
 
 from ...models import ObjectType
 
+from html.parser import HTMLParser
+
+
+class HTMLStripper(HTMLParser):
+    def __init__(self):
+        self.reset()
+        self.strict = False
+        self.convert_charrefs= True
+        self.fed = []
+    def handle_data(self, d):
+        self.fed.append(d)
+    def get_data(self):
+        return ''.join(self.fed)
+
+def strip_tags(html):
+    s = HTMLStripper()
```

Comment:
Can you use lxml instead? You will need to add it to dependencies.

http://stackoverflow.com/questions/753052/strip-html-from-strings-in-python#comment6232308_753083